Bottom: 5790c87173be93445f3c39a94780d9359764d5dc
Top:    bde09f47e850936d8537ab213ca1964b4d6db558
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-20 02:39:02 +0000

Refresh of crasy-test

---

diff --git a/src/draw.c b/src/draw.c
index 8e8cba2..bfa7ebd 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -299,7 +299,7 @@ draw_element_package (ElementType *element)
   /* draw lines, arcs, text and pins */
   ELEMENTLINE_LOOP (element);
   {
-    //_draw_line (line);
+    //_draw_line (Output.fgGC, line);
     dapi->draw_line (line, NULL, NULL);
   }
   END_LOOP;
